src/wtc-gallery-component.js

```diff
@@ -90,6 +107,47 @@ class Gallery extends ElementController {
     }
   }
 
+  /**
```

Comment:
Can we update all the documentation to jsdoc? See https://github.com/wethegit/wtc-sticker-book/blob/master/src/js/application/Stage.js for examples

Reply:
Updated. Side note: In looking through the JSdoc docs, I found that you don't need to specify certain things for ES2015 classes—@constructor and @method, for example, are picked up automatically!

Reply:
nice!